Tetris Clone C++/TetrisPiece.cpp

```diff
@@ -241,24 +238,20 @@ void TetrisPiece::setShapes() {
     this->bbox.setFillColor(sf::Color(0xFFFFFF7F));
     this->bbox.setOutlineColor(sf::Color(0xFFFFFFFF));
     this->bbox.setOutlineThickness(3);
-    this->printGrid();
+    //this->printGrid();
 }
 
 void TetrisPiece::printGrid() {
     int max = 4*4;
-    // std::cout << "TetrisPiece::printGrid() start" << std::endl << "printing grid" << std::endl;
-    std::cout << "+----+" << std::endl;
     for (int i = 0; i < max; i++) {
         if (i % 4 == 0) {
-            std::cout << '|';
+    
```

Comment:
These lines suggest a batch find/replace happened and had unintended consequences.